KubeArmor/monitor/processTree.go

```diff
@@ -173,6 +173,23 @@ func (mon *SystemMonitor) GetCommand(containerID string, hostPid uint32) string
 	ActivePidMapLock.Lock()
 	defer ActivePidMapLock.Unlock()
 
+	if pidMap, ok := ActiveHostPidMap[containerID]; ok {
+		if node, ok := pidMap[hostPid]; ok {
+			return node.Source
+		}
+	}
+
+	return ""
+}
+
+// GetCommandWithArgs Function
+func (mon *SystemMonitor) GetCommandWithArgs(containerID string, hostPid uint32) string {
+	ActiveHostPidMap := *(mon.ActiveHostPidMap)
+	ActivePidMapLock := *(mon.ActivePidMapLock)
+
+	ActivePidMapLock.Lock()
+	defer ActivePidMapLock.Unlock()
+
```

Comment:
I didn't understand this change!! What does this achieve?
New function GetCommandWithArgs() is introduced but not used!

Reply:
Yes @nyrahul 
From the existing function GetCommand(), we removed adding arguments to source.
If anyone requires to use args to source, then this function is introduced (GetCommandWithArgs).

It is currently unused. It is just kept for future use if needed.

Reply:
I see. Thanks